lib/routes.js

```diff
@@ -70,6 +70,8 @@ appRouter.get('/s/:shortid/:action', response.publishNoteActions)
 appRouter.get('/p/:shortid', response.showPublishSlide)
 // publish slide actions
 appRouter.get('/p/:shortid/:action', response.publishSlideActions)
+// gey my note list
+appRouter.get('/myNotes', noteController.listMyNotes)
```

Comment:
Maybe we can use like `/api/notes` instead of root level path to avoid occupied more routes when introduce more APIs.

Reply:
Yes we should do this. @JamesCamel Please change the route to `/api/notes/myNotes` 👍

Reply:
Fixed.